resolvers/nip11.ts

```diff
@@ -0,0 +1,51 @@
+export type RelayInformation = {
+    name?: string;
+    description?: string;
+    pubkey?: string;
+    contact?: string;
+    supported_nips?: number[];
+    software?: string;
+    version?: string;
+    icon?: string;
+};
+
+export const not_modifiable_information: {
```

Comment:
Why export?

Reply:
Because it needs to be used in `test.ts`.